apps/web/src/pages/confirmation.tsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect } from 'react';
+import { withRouter, NextRouter, useRouter } from 'next/router';
+import { ExternalLink, Notice, Button } from '@components';
+
+interface WithRouterProps {
+  router: NextRouter;
+}
+
+const Confirmation: React.FC<WithRouterProps> = props => {
+  const nextRouter = useRouter();
+  const { router } = props;
+  const id = router?.query?.id;
```

Comment:
I don't love that this page can be access by simply filling in an ID manually, this could arguably be better as react state but I don't think it's a merge blocker @fwpushan @dbayly-freshworks what do you think of storing the ID in the url?

Reply:
There might be a case for this being a sensitive code as it can be used to retrieve PII and storing it in a URL might not be the best option

Reply:
even though id is passed as a query param, it is null if the page is not redirected by router.